CMakeLists.txt

```diff
@@ -331,6 +343,51 @@ install(FILES "${CMAKE_CURRENT_BINARY_DIR}/g2.dat" DESTINATION share/${PROJECT})
 install(DIRECTORY data/ DESTINATION share/${PROJECT})
 install(FILES ${DOC_FILES} DESTINATION share/doc/${PROJECT})
 
+INCLUDE(ExternalProject)
+# Add gtest
+# http://stackoverflow.com/questions/9689183/cmake-googletest
+ExternalProject_Add(
+		googletest
+		URL https://github.com/google/googletest/archive/release-1.7.0.zip
```

Comment:
1.8.0 has a subdirectory for the `googletest` project, found it hard to find docs on how to get that to work. 1.8 is welcome though, as that would make it possible to give names to parameterized testcases